pkgs/by-name/atomic-browser/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  fetchFromGitHub,
+  lib,
+  mkPnpmPackage,
+}: let
+  inherit
+    (lib)
+    licenses
+    maintainers
+    ;
+in
+mkPnpmPackage rec {
+  pname = "atomic-browser";
+  version = "v0.37.0";
+
+  monorepoSrc = fetchFromGitHub {
+    owner = "atomicdata-dev";
+    repo = "atomic-server";
+    rev = "v0.37.0";
+    hash = "sha256-+Lk2MvkTj+B+G6cNbWAbPrN5ECiyMJ4HSiiLzBLd74g=";
+  };
```

Comment:
there's one other package with a monorepo here, should we follow the same pattern @Chickensoupwithrice 

https://github.com/ngi-nix/ngipkgs/blob/2aaa8b564709560bcb2add226627dd7e49562b31/pkgs/by-name/mcaptcha/package.nix#L12-L17

Reply:
Seems good enough to follow the convention